tests/integ/test_lineage.py

```diff
@@ -247,6 +246,37 @@ def test_lineage_trace(session):
         "DISTANCE": [1, 2],
     }
 
+    expected_df = pd.DataFrame(expected_data)
+    # TODO Enable the check after SNOW-1437475 is fixed.
+    # assert_frame_equal(df, expected_df, check_dtype=False)
+
+    # CASE 7 : Column lineage
+    session.sql(
+        f'CREATE OR REPLACE VIEW {db}.{schema}."v7" AS SELECT * FROM {db}.{schema}.V2'
```

Comment:
shall we cover a case where column project happens here?

Reply:
Do you mean "v7".C1 ?

Reply:
select cols... from

Reply:
changed the definition of the view to use projection.